src/components/organisms/AdminVenueView/components/PrettyLink/PrettyLink.scss

```
@@ -0,0 +1,14 @@
+@import "scss/constants.scss";
+
+.PrettyLink {
+  display: inline-block;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  color: opaque-white(0.5);
+  cursor: pointer;
+
+  & * {
```

shouldn't it work withou `&`? and are we sure we want to use `*` here instead of some class name? 
```suggestion
  * {
```

---

using `&` as a parent container limits the `*` scope which makes it "sure" enough